source/class/cv/TemplateEngine.js

```diff
@@ -460,6 +484,16 @@ qx.Class.define('cv.TemplateEngine', {
       }
       if (pagesNode.getAttribute('backend-url') !== null) {
         settings.backendUrl = pagesNode.getAttribute('backend-url');
+        window.console.error('The useage of "backend-url" is depreciated. Please use "backend-knxd-url", "backend-mqtt-url" or "backend-openhab-url" instead.');
```

Comment:
Please use build-in logging mechanism `this.error` or `qx.log.Logger.error` in static functions to be in sync with all other logging in the app and not this construct to please the linter.